sycl/test-e2e/ESIMD/unified_memory_api/prefetch_acc_dg2_pvc.cpp

```diff
@@ -0,0 +1,39 @@
+//==------- prefetch_acc_dg2_pvc.cpp - DPC++ ESIMD on-device test -------==//
```

Comment:
Please also create 'stateless' variant of the test, or even better: rename this one to `prefetch_acc_stateful_dg2_pvc.cpp` and create `prefetch_acc_dg2_pvc.cpp` that would be compiled with default switches (in stateless mode).